frontend/src/main/java/com/google/tripmeout/frontend/TripModel.java

```diff
@@ -8,6 +8,7 @@ public static Builder builder() {
     return new AutoValue_TripModel.Builder();
   }
 
+  public abstract Builder toBuilder();
```

Comment:
Nit: please move this below the TripModel fields and add a separating space. This makes it clear that toBuilder is not semantically part of this value class.